### DISCUSSION TITLE
ETL in v3.5.9

### DISCUSSION THREAD

**Discussion by roklenardic** [ravendb/ravendb#15606] [General]
Hi,

I am trying to test the ETL functionality of the RavenDB in version 3.5.9 but cannot get it to work. 
I did the following:
1. Backed up source DB
2. Restored source backup into destination DB
3. Enabled the ETL functionality in the source DB, targetting the destination DB. I also added the Orders collection to he 
4. Add the Orders to the ETL collections
5. Added another json to the Orders collection in source database
6. Cannot see anything in the destination DB

Could it be that I somehow need to enable ETL as a bundle? The ETL screen tells me that replication is disabled, is that required for ETL to work?
![image](https://user-images.githubusercontent.com/121102/211822128-8b16b622-679d-49a5-a5e2-743a77b3c486.png)

Looking at the Database Settings of the source db it looks as if the Replication bundle is enabled?
![image](https://user-images.githubusercontent.com/121102/211822988-868760fe-e4b2-4698-b4aa-e132564253a2.png)

Any tips greatly appreciated, the RavenDB ETL docs are very skimpy on this...

**Comment by grisha-kotler** [comment]
IIRC, the replication bundle should be enabled on both the source and destination.

Note that this version is out of support.

**Reply by roklenardic** [reply]
Thanks @grisha-kotler , not sure how I could enable ETL on the destination as you can only add destinations to the ETL screen. Obviously I don't want an ETL back to source...

I am aware that 3.5.x is out of support but still need to figure out some prod-stage-dev ETL solution in a client's environment that uses it.

**Reply by ayende** [reply]
ETL & Replication are two different things, FWIW.
For ETL, you only need that enabled in the source.

**Reply by roklenardic** [reply]
Yeah, I have that but for some reason nothing happens.

**Reply by ayende** [reply]
Anything further would require opening a support call, I'm afraid. That is applicable for users with extended support contract for 3.5
